platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/SerializationTests.java

```diff
@@ -53,20 +51,7 @@ public <T extends SelfSerializable> void serializeDeserializeTest(T generated) t
     }
 
     static Stream<Arguments> selfSerializableProvider() {
```

Comment:
Is it still necessary to have the parameterized test, now that the parameter stream only has 1 element?

Reply:
I don't know who wrote the class, but it looks like the idea was for it to work with any kind of serializable object, its just that this is the only class that was added. I would leave it like this, but I can change it if you feel strongly about it